lib/jerboa/format/body/attribute/lifetime.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Jerboa.Format.Body.Attribute.Lifetime do
+  @moduledoc """
+  LIFETIME attribute as defined in [TURN RFC](https://trac.tools.ietf.org/html/rfc5766#section-14.2)
+  """
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Lifetime.LengthError
+
+  defstruct duration: 0
+
+  @max_duration :math.pow(2, 32) - 1
```

Comment:
It's the maximum that fits into 32 bits of a binary. I know that we don't need constraint because binary constructor would handle overflows, but I thought it would be nice to at least raise an error if someone passes wrong value (which probably won't happen, because the number is huge).